packages/webapp/pages/_app.tsx

```diff
@@ -78,15 +80,17 @@ function InternalApp({ Component, pageProps, router }: AppProps): ReactElement {
 
   useEffect(() => {
     if (
-      isOnboardingReady &&
+      isOnboardingActionsReady &&
       (!hasCompletedEditTags || !hasCompletedContentTypes) &&
       !router.pathname.includes('/onboarding')
     ) {
       router.replace('/onboarding');
     }
+
+    // @NOTE see https://dailydotdev.atlassian.net/l/cp/dK9h1zoM
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [
-    isOnboardingReady,
-    router,
```

Comment:
Router has been known to an unstable dependency, hence removed.

Reply:
its not unstable since we upgraded to next 14 code, it no longer changes on each render, it only changes when page changes, which is supposed to happen

Reply:
Ah, finally. It is quite annoying to have them commented in the past.